src/components/OnlyCard/OnlyCard.js

```diff
@@ -0,0 +1,22 @@
+import { Fragment, useState } from "react";
+import { CardContent } from "../CardContent/CardContent";
+import CardImage from "../CardImage/CardImage";
+import Card from "../Card/Card";
+
+const OnlyCard = ({ items }) => {
+  const [isHovered, setIsHovered] = useState(false);
+  const handleMouseOver = () => setIsHovered(true);
+  const handleMouseLeave = () => setIsHovered(false);
+  return (
+    <Fragment>
+      <a href={items} target="_blank" rel="noopener noreferrer">
```

Comment:
P1:
items 객체 자체를 넘겨주면 이동이 안 됩니다.
```suggestion
      <a href={items.url} target="_blank" rel="noopener noreferrer">
```